sdcm/nemesis.py

```diff
@@ -370,7 +472,16 @@ def _set_current_disruption(self, label=None, report=True, node=None):
         self.current_disruption = label
         self.log.info(label)
         if report:
-            DisruptionEvent(type='start', name=self.get_disrupt_name(), status=True, node=str(current_node))
+            self._issue_disruption_start_event(current_node)
+
+    def _issue_disruption_start_event(self, node=None):
```

Comment:
It's not clear why those functions are introduced

Reply:
need to remove